### PR TITLE
Add a vagrant playbook for local testing

### PR DESCRIPTION
**Pull request by JackWilb**
Based on our discussion about local testing becoming harder with the ssl tasks.

### CODE REVIEW

**Comment by jjnesbitt** [comment]
@JackWilb thanks for fixing the `server_tasks`, I'll have to run this in vagrant locally and take a look.

**Comment by JackWilb** [comment]
@AlmightyYakob Yeah, should be good to go. I figured I'd leave the commented out code so we can see the differences. If that's not best practices/ you think it looks ugly, let me know

File: playbook.vagrant.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    ansible_python_interpreter: /usr/bin/python3
+    multinet_server_path: "{{ ansible_env.HOME }}/multinet"
+    adjmatrix_client_path: "{{ ansible_env.HOME }}/adjmatrix"
+    adjmatrix_public_path: "adjmatrix"
+    nodelink_client_path: "{{ ansible_env.HOME }}/nodelink"
+    nodelink_public_path: "nodelink"
+    arangodb_password: "letmein"
+    lockfile_path: /tmp/ansible-playbook.lock
+    force_ignore_lock: false
+    # Modified for local build
```
Comment (jjnesbitt) [inline playbook.vagrant.yml]:
```suggestion
```

Don't think this is necessary, since this file is only used for local builds.

**Review by jjnesbitt — APPROVED** [review]
LGTM, works on my local, just left a small suggestion.

**Pull request merged**